docs/providers.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Providers
+toc: true
+weight: 400
+indent: true
+---
+
+# Providers
+
+Providers are external infrastructure providers such as Amazon Web Services,
+Microsoft Azure, Google Cloud and Alibaba Cloud.
+
+In order to provision a resource in the provider, a Custom Resource Definition(CRD)
+needs to be registered in your Kubernetes cluster and its controller should
+be watching instances of that Custom Resource Definition. Provider packages
+contain many Custom Resource Definitions and their controllers to extend
+functionality of Crossplane.
+
+Here is the list of current providers:
+
+* [Amazon Web Services][provider-aws]
+  * [API Reference][aws-reference]
+* [Google Cloud][provider-gcp]
+  * [API Reference][gcp-reference]
+* [Microsoft Azure][provider-azure]
+  * [API Reference][azure-reference]
+* [Rook][provider-rook]
+  * [API Reference][rook-reference]
+* [Alibaba Cloud][provider-alibaba]
+  * [API Reference][alibaba-reference]
+
+## Installation
+
+Since all managed resources are all cluster-scoped, provider packages are
+bundled as `ClusterPackage`s. In order to request installation of a provider,
```

Comment:
This might be confusing if you don't know what a ClusterPackage is. Consider just introducing the `ClusterPackageInstall` type like:
The core Crossplane controller can install Provider controllers and CRDs for you through its own provider packaging mechanism, which is triggered by the application of a `ClusterPackageInstall` resource. For example, in order to request installation of the `provider-gcp` package, apply the following resource to the cluster where Crossplane is running: